lib/routes.dart

```diff
@@ -224,6 +234,10 @@ final List<RouteBase> routes = [
       ),
     ],
   ),
+  GoRoute(
+    path: '/members/photos/liked-photos',
```

Comment:
It would be nice to have /members/photos/liked also as a route as thats what its like on the website